### ISSUE TITLE
Preserve Natural Scrolling 

### ISSUE THREAD

**Issue by claytron5000** [pop-os/pop#1551]
**Distribution (run `cat /etc/os-release`):**
 NAME="Pop!_OS"
VERSION="20.04 LTS"
ID=pop
ID_LIKE="ubuntu debian"
PRETTY_NAME="Pop!_OS 20.04 LTS"
VERSION_ID="20.04"
HOME_URL="https://pop.system76.com"
SUPPORT_URL="https://support.system76.com"
BUG_REPORT_URL="https://github.com/pop-os/pop/issues"
PRIVACY_POLICY_URL="https://system76.com/privacy"
VERSION_CODENAME=focal
UBUNTU_CODENAME=focal
LOGO=distributor-logo-pop-os

**Related Application and/or Package Version (run `apt policy $PACKAGE NAME`):**

**Issue/Bug Description:**
Natural Scrolling is reset every time I wake the computer from sleep. Although the UI does not reflect this.

**Steps to reproduce (if you know):**
- Toggle on Natural Scrolling from Settings
- Close lid, or otherwise sleep the computer
- Open computer and us scroll wheel on mouse.

**Expected behavior:**
- Expected behaviour: "Natural Scroll" action is preserved.
- Actual behaviour:  "Reverse Scroll" action happens

**Other Notes:**
Note: The UI presents the Natural Scroll as toggled on. However, toggling this off then on again gets back to actual Natural Scroll behaviour.

**Comment by colinodell** [comment]
Possibly related to:

 - https://github.com/pop-os/shell/issues/598
 - https://github.com/pop-os/gnome-control-center/issues/120
 - https://gitlab.gnome.org/GNOME/mutter/-/issues/1505
 - https://askubuntu.com/questions/1284154/natural-scrolling-setting-get-reset
 - https://askubuntu.com/questions/1286559/scroll-direction-keeps-changing-after-recent-upgrade

**Comment by bflanagin** [comment]
Based on this [issue](https://gitlab.gnome.org/GNOME/gnome-shell/-/issues/3421) over on gitlab. It seems to be fixed in gnome 3.38.2+ and mutter 3.38.2+ 

@claytron5000: Could you run the commands below and post there output here.

`apt policy gnome-shell`

and

`apt policy mutter`

**Comment by claytron5000** [comment]
```
> apt policy gnome-shell
gnome-shell:
  Installed: 3.36.4-1ubuntu1~20.04.2pop1~1597874448~20.04~260a1a7
  Candidate: 3.36.4-1ubuntu1~20.04.2pop1~1597874448~20.04~260a1a7
  Version table:
 *** 3.36.4-1ubuntu1~20.04.2pop1~1597874448~20.04~260a1a7 1001
       1001 http://ppa.launchpad.net/system76/pop/ubuntu focal/main amd64 Packages
        100 /var/lib/dpkg/status
     3.36.4-1ubuntu1~20.04.2 500
        500 http://us.archive.ubuntu.com/ubuntu focal-security/main amd64 Packages
        500 http://us.archive.ubuntu.com/ubuntu focal-updates/main amd64 Packages
     3.36.1-5ubuntu1 500
        500 http://us.archive.ubuntu.com/ubuntu focal/main amd64 Packages
```
```
apt policy mutter
mutter:
  Installed: 3.36.6-1ubuntu0.20.04.2pop1~1601410113~20.04~f0bfd9b
  Candidate: 3.36.6-1ubuntu0.20.04.2pop1~1607379824~20.04~5a3d5e6
  Version table:
     3.36.7+git20201123-0.20.04.1 500
        500 http://us.archive.ubuntu.com/ubuntu focal-updates/main amd64 Packages
     3.36.6-1ubuntu0.20.04.2pop1~1607379824~20.04~5a3d5e6 1001
       1001 http://ppa.launchpad.net/system76/pop/ubuntu focal/main amd64 Packages
 *** 3.36.6-1ubuntu0.20.04.2pop1~1601410113~20.04~f0bfd9b 100
        100 /var/lib/dpkg/status
     3.36.1-3ubuntu3 500
        500 http://us.archive.ubuntu.com/ubuntu focal/main amd64 Packages
```

**Comment by jacobgkau** [comment]
@claytron5000 Your output shows that `mutter` is out of date on your machine. Please install all updates and reboot using these commands:
```
sudo apt update
sudo apt full-upgrade
sudo apt autoremove --purge
sudo apt install pop-desktop
sudo systemctl reboot
```
You can see in the git history [here](https://github.com/pop-os/mutter/commits/master_focal) that the version you have installed (`f0bfd9b`) is one version before the fix for this specific issue; the version that is available (`5a3d5e6`) is the version with the fix. Please let us know if the issue continues after you're up-to-date.

---

System76 customers can reach out to [support](https://support.system76.com/) for technical assistance. For non-System76 hardware, you can seek community support on [Reddit](https://reddit.com/r/pop_os/) or [Mattermost](https://chat.pop-os.org).

**Comment by jacobgkau** [comment]
I'm closing this issue because it was fixed. If the issue reappears with an up-to-date Mutter version, please let us know and we can re-open.
